resource/resource.go

```diff
@@ -22,6 +22,6 @@ package resource
 // When changing files included here, you may add the `-debug` flag, which will
 // avoid embedding any files (but instead passes them through from disk). This
 // avoids having to recompile before testing changes.
-//go:generate go-bindata -pkg resource -mode 0644 -modtime 1400000000 -o ./embedded.go ./ui/...
+//go:generate go-bindata -pkg resource -mode 0644 -modtime 1400000000 -o ./embedded.go ./ui/... ./test_certs/...
```

Comment:
I'd like to embed the certs in a separate package that is only imported from tests. I don't want release binaries to contain the test certs since it would be a bad idea to ever actually use them.